app/controllers/time_summary_controller.rb

```diff
@@ -178,14 +194,14 @@ def time_summary_data_csv
         (@start_date..@end_date).each do |date|
           if date.workday?
             if users[user.user][date].present?
-              row << users[user.user][date]
+              users[user.user][date].each { |tc| row << tc }
```

Comment:
```suggestion
              row += users[user.user][date]
```
Debería ser lo mismo, igual sino ya con el cambio del map es un _avance_.

Reply:
Muchas gracias, ahora lo pruebo

Reply:
como si supieras!!! Gracias